modules/decision/landing_pad_tracking.py

```diff
@@ -82,16 +86,16 @@ def run(
             # If new landing pad, add to list of unconfirmed positives
             self.__unconfirmed_positives.append(detection)
 
-        # If there are confirmed positives, return the first one
+        # If there are confirmed positives, return them
         if len(self.__confirmed_positives) > 0:
-            return True, self.__confirmed_positives[0]
```

Comment:
this changes the behaviour of decision module right?

Reply:
Good question, are there any other modules using this? If not then I think it looks good

Reply:
Yes, this is the fix.